scripts/submit_skims.sh

```diff
@@ -206,7 +206,7 @@ REGEX_MAP=(
     ["TTTo2L2Nu"]="TT_FullyLep"
     ["TTToSemiLeptonic"]="TT_SemiLep"
 
-    ["DYJets.+_M-50_T.+amc"]="DY_Incl"
+    ["DYJetsToLL_M-50_TuneCP5_13TeV-amc"]="DY_Incl"
```

Comment:
same bug affects `    ["WJetsToLNu_T.+madgraph"]="WJets_HT0To70" # for 0 < HT < 70`

Reply:
Thanks for pointing it out. It is handled in #363.